client/src/containers/BBLPage.tsx

```diff
@@ -159,11 +159,11 @@ export default class BBLPage extends Component<BBLPageProps, State> {
       return (
         <Redirect
           to={{
-            pathname: createRouteForAddressPage(
-              addressForURL.boro,
-              addressForURL.streetname,
-              addressForURL.housenumber
-            ),
+            pathname: createRouteForAddressPage({
+              boro: addressForURL.boro,
+              streetname: addressForURL.streetname,
+              housenumber: addressForURL.housenumber,
+            }),
```

Comment:
err is there a reason you're not just passing `addressForURL` here, e.g. `createRouteForAddressPage(addressForURL)`?  That would make it a lot more concise...